vignettes/email-overview.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+title: "Überblick Open-Access-Metadaten-Schnelltest"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Überblick Open-Access-Metadaten-Schnelltest}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Überblick Open-Access-Metadaten-Schnelltest
+===========================================
+
+Überblick
+---------
+
+Umfangreiche Metadaten sichern die Auffindbarkeit und Nachnutzung von
+Open-Access-Artikeln. Aus diesem Grunde ist die Überprüfung der
+Metadaten ein wesentlicher Workflow-Bestandteil beim Umgang mit
+Open-Access-Publikationskosten und Transformationsverträgen.
+
+Der Open-Access-Metadaten-Schnelltest ermöglicht die automatische
+Überprüfung der Metadatenqualität von Open-Access-Zeitschriftenartikeln.
+Er richtet sich in erster Linie an Bibliotheken und wissenschaftliche
+Informationseinrichtungen, die Open-Access-Publikationskosten etwa im
+Rahmen eines Publikationsfonds oder eines Transformationsvertrags
+verwalten.
+
+Im Mittelpunkt der Überprüfung des  Open-Access-Metadaten-Schnelltests 
+stehen die Metadaten der DOI-Registrierungsagentur Crossref, welche von den 
+meisten wissenschaftlichen Verlagen als Metadatenservice genutzt wird. Daher
+greifen vielfältige Forschungsvorhaben, Discovery- und
+Data-Analytics-Serivces auf die Crossref-Datendienste zu, um
+Open-Access-Artikel zu finden und nachzunutzen.
+
+Testergebnisse
+--------------
+
+Das Testergebnis des Open-Access-Metadaten-Schnelltest wird per Email
+verschickt und beinhaltet folgende Ergebnisse:
+
+### Indexierungsstatus in Crossref
+
+Der Abschnitt fasst zusammen, für welche DOIs keine Metadaten in
+Crossref verfügbar sind. Es kann vielfältige Ursachen haben, warum eine DOI
+nicht in Crossref indexiert ist:
+
+-   DOI ist syntaktisch fehlerhaft
+
+-   DOI ist vom Verlag bereits registriert, der Artikel ist jedoch noch
+    nicht publiziert
+
+-   DOI wurde nicht bei Crossref, sondern bei einer anderen Agentur
+    registriert
+
+### Überblick: Crossref-Metadaten
+
+Der Abschnitt zeigt einen tabellarischen Überblick der Testergebnisse.
+Es werden absolute und relative Häufigkeiten dargestellt.
+
+Untersucht wurden:
+
+-   Verfügbarkeit von Creative-Commons-Lizenzen (CC License)
+
+-   Verfügbarkeit von regelkonformen CC-Lizenzen (Compliant CC)
+
+-   Unterstützung von Versionen für das Text- und Data-Mining (TDM) (TDM
+    Support)
+
+-   Vorhandensein von Förderinformationen (Funder info)
+
+-   Verknüpfung zwischen Publikationen und ORCID
+
+-   Unterstützung weitere offener Metadaten 
+    ([Open Abstracts](https://i4oa.org/) 
+    und [Open Citations](https://i4oc.org/))
+
+### Überblick Lizenzbedingungen
+
+Offene Lizenzen sind wesentlich, um den Zugang und die Nachnutzung von
```

Comment:
we should always have one line per sentence (or other punctuation), for meaningful diffs.
This also appears to be the guiding practice for markdown syntax, see https://github.com/tidyverse/style/issues/162 for details.

Reply:
will fix in a sec.